tests/sagemaker/scripts/tensorflow/run_tf_dist.py

```diff
@@ -6,12 +6,26 @@
 
 import tensorflow as tf
 from datasets import load_dataset
+from packaging.version import parse
 from tqdm import tqdm
 
 from transformers import AutoTokenizer, TFAutoModelForSequenceClassification
 from transformers.utils import is_sagemaker_dp_enabled
 
 
+try:
```

Comment:
Why use the try/except here instead of `from transformers.modeling_tf_utils import keras`?

Reply:
Fixed!